contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -21,21 +17,23 @@ import { Errors } from "../../../lib/Errors.sol";
 /// @title Ip Royalty Vault
 /// @notice Defines the logic for claiming revenue tokens for a given IP
 /// @dev [CAUTION]
-///      Do not transfer ERC20 tokens directly to the ip royalty vault as they can be lost if the pendingVaultAmount
+///      Do not transfer ERC20 tokens directly to the ip royalty vault as they can be lost if the poolInfo
 ///      is not updated along with an ERC20 transfer.
-///      Use appropriate callpaths that can update the pendingVaultAmount when an ERC20 transfer to the vault is made.
-contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, ReentrancyGuardUpgradeable {
+///      Use appropriate callpaths that can update the poolInfo when an ERC20 transfer to the vault is made.
+contract IpRoyaltyVault is IIpRoyaltyVault, ERC20Upgradeable, ReentrancyGuardUpgradeable {
     using EnumerableSet for EnumerableSet.AddressSet;
-    using SafeERC20Upgradeable for IERC20Upgradeable;
+    using SafeERC20 for IERC20;
 
     /// @dev Storage structure for the IpRoyaltyVault
     /// @param ipId The ip id to whom this royalty vault belongs to
-    /// @param lastSnapshotTimestamp The last snapshotted timestamp
-    /// @param pendingVaultAmount Amount of revenue token pending to be snapshotted
-    /// @param claimVaultAmount Amount of revenue token in the claim vault
-    /// @param claimableAtSnapshot Amount of revenue token claimable at a given snapshot
-    /// @param isClaimedAtSnapshot Indicates whether the claimer has claimed the revenue tokens at a given snapshot
+    /// @param lastSnapshotTimestamp [DEPRECATED] The last snapshotted timestamp
```

Comment:
Do we need [DEPRECATED] vs eliminate? Ideally we deploy cleanly at the start

Reply:
Noticed that the variables were eliminated but the natspec still seems to have the [DEPRECATED] variables?

Reply:
Got it, we can add an issue to remove all [DEPRECATED] variables for fresh deployment